Source/WebCore/platform/network/ResourceHandle.cpp

```diff
@@ -265,24 +273,24 @@ void ResourceHandle::setNetworkLoadMetrics(Box<NetworkLoadMetrics>&& metrics)
     d->m_networkLoadMetrics = WTFMove(metrics);
 }
 
-bool ResourceHandle::shouldContentSniff() const
+ContentSniffingPolicy ResourceHandle::contentSniffingPolicy() const
 {
-    return d->m_shouldContentSniff;
+    return d->m_contentSniffingPolicy;
 }
 
 ContentEncodingSniffingPolicy ResourceHandle::contentEncodingSniffingPolicy() const
 {
     return d->m_contentEncodingSniffingPolicy;
 }
 
-bool ResourceHandle::shouldContentSniffURL(const URL& url)
+ContentSniffingPolicy ResourceHandle::contentSniffingPolicyForURL(const URL& url)
 {
 #if PLATFORM(COCOA)
     if (shouldForceContentSniffing)
```

Comment:
This shouldForceContentSniffing stuff is leftover from WebKitLegacy and only set to true behind a linked on or after (or I guess a linked on or before) check.